### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by 1ozturkbe** [hoburg/d8#70]
I think it is time to start talking about documentation. @bqpd I would like to talk to you at some point about creating a readthedocs so that people can make use of our tool. Would you be free to meet today sometime between 1-4PM?

**Comment by 1ozturkbe** [comment]
I guess this has passed, but I have started work on the docs in the readthedocs branch.

**Comment by bqpd** [comment]
Taking a look :D

**Comment by bqpd** [comment]
Looks like the branch name is readthedocs-1 on github?

**Comment by 1ozturkbe** [comment]
Yeah. As I said, I wanted to meet to be able to get some pointers to get over the learning curve. But I think I can handle it. If you have any good resources other than the documentation for readthedocs, that would be appreciated. I am going to go through the tutorials otherwise.

**Comment by 1ozturkbe** [comment]
As an update, documentation has started in the docs branch.

**Comment by mayork** [comment]
just took a look at this...great work @1ozturkbe !

**Comment by 1ozturkbe** [comment]
Thanks! It is a good start, but it will be a lot nicer once we have documented all of the submodels and streamlined the code.

**Comment by 1ozturkbe** [comment]
@mayork I will start updating the documentation with the contents of the two papers. Do you mind if I push all things docs related directly to master?

**Comment by mayork** [comment]
Nah that's good

**Comment by 1ozturkbe** [comment]
@bqpd I am hating readthedocs so much... is there any good documentation or tutorials that you know of? Why is it such a pain to update anything?

**Comment by bqpd** [comment]
huh, for me that was the easiest part...you just have to add a hook to your repo, right?

**Comment by bqpd** [comment]
oh wait I was wrong about that, it has some other way to listen...well, take a look at https://readthedocs.org/projects/*/builds/ (where * is your repo name) to make sure it's building?

**Comment by 1ozturkbe** [comment]
i did get it working... except for citations. may ask you some basic questions at team meeting

**Comment by 1ozturkbe** [comment]
i was having issues because i was trying to shift to new url.
